### ISSUE TITLE
MQTT as datasource

### ISSUE THREAD

**Issue by Rello** [Rello/analytics_sourcepack#1]
https://github.com/Rello/analytics/issues/90

@roadfox
your topic will be tracked here.
a separate datasource collection will be offered to deal with special datasources

**Comment by htot** [comment]
Yeah this would be very nice and allow analytics to visualize data from fleets of devices that they publish to their MQTT server. That could be a https://www.hivemq.com/ or a local server (like mosquitto). The role of analytics would be similar to MQTT Explorer but then to build a dashboard.
